test/unit/configure-robot.test.ts

```diff
@@ -1,52 +0,0 @@
-import supertest from "supertest";
-import { createProbot } from "probot";
-import * as PrivateKey from "probot/lib/private-key";
-import configureAndLoadApp from "../../src/configure-robot";
-import {Writable} from "stream";
-
-describe('configure-robot', () => {
```

Comment:
Why is this deleted?

Reply:
It wasn't really testing anything that we aren't already testing in other places.  This was a remnant of the old code and didn't think it was needed anymore.